### ISSUE TITLE
FlatList/ScrollView use of flexGrow:1 to fill view is not documented

### ISSUE THREAD

**Issue by jamesxabregas** [facebook/react-native-website#1635]
I recently came across an issue where I wanted my FlatList to take up the full screen, but adding `flex:1` to the style properties made it disappear.

I found the answer to this by way of a [StackOverflow post](https://stackoverflow.com/questions/52286459/how-to-make-flatlist-fill-the-height) that lead me to the [Github thread on the issue](https://github.com/facebook/react-native/issues/4099) which instructed me to use the style property `flexGrow: 1` on the FlatList, with `flex: 1` on the surrounding components.

The problem is that this is not adequately documented. [FlatList](https://facebook.github.io/react-native/docs/flatlist) and [ScrollView](https://facebook.github.io/react-native/docs/scrollview) have no mention of flexGrow at all on their pages, and the [documentation for the style property for flexGrow](https://facebook.github.io/react-native/docs/layout-props#flexgrow) has no description of what it does whatsoever. At present it is impossible for someone to resolve the issue I encountered by reading the documentation alone.

**Comment by Simek** [comment]
The Flexbox guide can be found [here](https://reactnative.dev/docs/flexbox) and `flexGrow` property is described [there](https://reactnative.dev/docs/flexbox#flex-basis-grow-and-shrink). At the top of the [Layout Props](https://reactnative.dev/docs/layout-props) there is a mention that you can find detailed description of props in the Flexbox guide.

Would you like to create PR which adds briefly description to the `flexGrow` and maybe to other Layout Props without description? Also I think that we should emphasize the Flexbox guide mention by putting it in the note box.

**Comment by jamesxabregas** [comment]
Hi there, I've done as you've suggested and added the descriptions for flexGrow, flexShrink, and flexBasis to the layout-props page. I will create a pull request. I have simply copied the descriptions from the Flexbox guide so that at least the Layout Propos API page is complete. I'm not sure what you meant by the last line where you said "Also I think that we should emphasize the Flexbox guide mention by putting it in the note box" so I haven't actioned that.

That said, I still don't think that this adequetly resolves the issue that I highlighted, which is that there is a gotcha with FlatList and ScrollView that is not adequetly documented. I'm still a newbie to React Native so I'm not sure the best way to progress.
